### ISSUE TITLE
parallax loading log meta

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#14342]
## Description
client logs show parallax loading messages during certain gamemodes
has hyper meta consequences

**Reproduction**
1. start certain gamemode
2. `
3. you now have meta knowledge

**Comment by 0x6273** [comment]
I guess the issue here is that all maps are sent to clients, rather than just the one they are on? Just removing the log messages wouldn't fix the problem, since the information is still sent to clients.

**Comment by deltanedas** [comment]
yeah there's no need to send every map to clients unless they are ftl/ghost warping

**Comment by ThatGuyUSA** [comment]
Solved by #27127

surely...

**Comment by VerinSenpai** [comment]
Correct me if Im wrong but... That pr is changing client side code. Wouldn't a modified client still get those messages and possess the great meta knowledge. Or at that point is it just not a concern.

**Comment by ThatGuyUSA** [comment]
Under rule 1.8 "Do not exploit the game, use cheats, or macros"

The following are prohibited by this rule:
4. anything which results in gaining elevated privileges, including admin permissions,
5. external tools, and client modifications, including macros, and
6. anything which prevents another player who is not game banned... etc...

I guess, but you shouldn't be doing it. I just close the issues!

**Comment by ThatGuyUSA** [comment]
I'm doing digital archeology here, if I see a PR that closes the issue, I close it.

If I know something's been fixed, and I'm not at work. I'll provide media showing the issue has been resolved without a PR.

But seeing as I'm at work... 😺

**Comment by deltanedas** [comment]
this issue was mostly about parallax because you could/can out nukies gamemode by the parallax loading message, cheaters could probably find out somehow anyway

**Comment by VerinSenpai** [comment]
Fair enough. The pr does say it fixes trivial metagaming.

**Comment by 0x6273** [comment]
The client shouldn't be aware of which maps exists, so there's still an issue here. Could it be re-opened or should I make a new issue?

## New reproduction steps
1. Use a modded client, or external software to capture the network traffic.
2. Nukie gamemode on the server.
3. Now you can tell that there are nukies by checking if the nukie map has been sent from the server.

## Proposed fix
Server should not be leaking all the maps to clients. Only the map the player is in should be sent.

**Comment by ThatGuyUSA** [comment]
Open a new issue please!
